__tests__/actionUtils.test.ts

```diff
@@ -102,7 +102,8 @@ test("setOutputAndState with exact match to set cache-hit output and state", ()
     actionUtils.setOutputAndState(key, cacheKey);
 
     expect(setOutputMock).toHaveBeenCalledWith(Outputs.CacheHit, "true");
-    expect(setOutputMock).toHaveBeenCalledTimes(1);
+    expect(setOutputMock).toHaveBeenCalledWith(Outputs.CacheKey, key);
```

Comment:
Shouldn't it match to `cacheKey`?

Reply:
Shrug, `key` and `cacheKey` are the same.